tests/Providers/Identifiers/InnAndKppProviderTest.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace AvtoDev\FakerProviders\Tests\Providers\Identifiers;
+
+use AvtoDev\FakerProviders\Providers\Identifiers\InnAndKppProvider;
+use AvtoDev\FakerProviders\Tests\Providers\AbstractProviderTestCase;
+
+class InnAndKppProviderTest extends AbstractProviderTestCase
```

Comment:
Add `@coversDefaultClass` section in PhpDoc and add "@covers ::method" in tests section